src/styles.css

```
@@ -708,3 +708,50 @@ footer a:hover{text-decoration:underline}
+
+/* Dark Mode: Sun is hidden, Moon is visible */
+html.dark .att-svg .att-sun-icon { 
+  opacity: 0 !important; 
+  transform: scale(0.3) rotate(-90deg) !important; 
+}
+html.dark .att-svg .att-moon-icon { 
+  opacity: 1 !important; 
+  transform: scale(1) rotate(0deg) !important; 
+}
+
+html.theme-transition, html.theme-transition *, html.theme-transition *:before, html.theme-transition *:after { 
```

Stylelint (17.13.0)

[error] Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)

(selector-pseudo-element-colon-notation)